src/assets/styles/10-settings/_colours.scss

```diff
@@ -48,11 +48,11 @@
   --colour-red-90: #40120d;
   --colour-red-60: #e10f2d;
 
-  --link-colour: var(--colour-blue-60);
-  --link-colour-hover: var(--colour-blue-70);
-  --link-colour-focus-outline: var(--colour-blue-40);
-  --link-colour-active: var(--colour-blue-50);
-  --link-colour-disabled: var(--colour-grey-50);
+  --link-colour: var(--colour-grey-80);
```

Comment:
Ultimately we need to refactor and expand this set of variables to include background colours (which these were originally) but I think that's something to be tackled in the design system code.

Reply:
I will create a ticket to add variables for the button background-colours.